test/functional/admin/statistics_announcement_publications_controller_test.rb

```diff
@@ -69,7 +69,7 @@ class Admin::StatisticsAnnouncementPublicationsControllerTest < ActionController
 
     assert_response :success
     has_search_results_table
-    assert_select "ul.govuk-error-summary__list a[data-track-action='statistics announcement-error'][data-track-label=\"Publication type does not match: must be statistics\"]", text: "Publication type does not match: must be statistics"
```

Comment:
@minhngocd not sure why this was ever needed

Reply:
probably to test that we are sending that tracking data by setting those attributes

Reply:
It is tested elsewhere, the test is not about tracking, the test does not require tracking selector to pass.